org-roam-db.el

```diff
@@ -79,7 +79,7 @@ slow."
   :group 'org-roam)
 
 ;;; Variables
-(defconst org-roam-db-version 16)
+(defconst org-roam-db-version 17)
```

Comment:
Possibly unrelated to this PR, but I immediately noticed that building the database from scratch through automatic upgrade feels orders of magnitude more slower than building the database from scratch manually (i.e. through force sync). Not sure what's going on there.

Reply:
> Possibly unrelated to this PR, but I immediately noticed that building the database from scratch through automatic upgrade feels orders of magnitude more slower than building the database from scratch manually (i.e. through the force sync). Not sure what's going on there, but it's something reproducible.

Worth looking into separately, I guess.